### DISCUSSION TITLE
Goflight Pollypot (FCU PULl/Push) events

### DISCUSSION THREAD

**Discussion by studio051** [flybywiresim/aircraft#4773] [Help] [answered]
Hello I’m using Pollypot to map goflight units . The new version of Pollypot has a detective that finds LVARS helpful for assigning commands . 

The recent update on Github shows some new commands like FCU Push and Pull added but I don’t see them as events and none of the events are visible in the list also ? 

Any ideas on how to find and map them ?

**Comment by aguther** [comment, marked as answer]
Those events are no local variables (LVARs), those are custom SimConnect events. Both FSUIPC and SPAD.neXt support those kind of events. Please contact the support of PollyPot software to tell you if they support custom SimConnect events and if so how to configure them.

**Reply by studio051** [reply]
Thank you . Got the paid version of FSUIPC 7 . 

I can see many events and some are working fine but I'm interesting in mapping FCU altitude PULL and PUSH which I can't see in the FSUIPC list . 

Can you please advise me how to find those events in FSUIPC ? Anything I need to add to the folder ? What am I doing wrong ?

**Reply by aguther** [reply]
With FSUIPC it should be working fine (tested it myself). Please refer to the following page, section "Mapping of events to control autopilot / autothrust":
https://docs.flybywiresim.com/start/autopilot-fbw/

There is then a link to the FSUIPC event file I've created:
https://github.com/flybywiresim/a32nx/raw/master/docs/FSUIPC/flybywire-aircraft-a320-neo.evt

This needs to be put into the FSUIPC folder beside the configuration file (ini) of FSUIPC. Then you can use the events in button assignments of FSUIPC.
